app/helpers/test_controller_helper.rb

```diff
@@ -0,0 +1,2 @@
+module TestControllerHelper
+end
```

Comment:
To są pozostałości tego eksperymentu
  ----- Original Message ----- 
  From: Anna Ślimak 
  To: kraryk/test_blog 
  Cc: kraryk 
  Sent: Tuesday, January 26, 2016 3:17 PM
  Subject: Re: [test_blog] BŁąd przy wywoływaniu @post.tags (#3)


  In app/helpers/test_controller_helper.rb:

> @@ -0,0 +1,2 @@
> +module TestControllerHelper
> +end
do wywalenia

  —
  Reply to this email directly or view it on GitHub.